alvr/server_openvr/cpp/alvr_server/HMD.h

```diff
@@ -74,4 +74,8 @@ class Hmd : public TrackedDevice, vr::IVRDisplayComponent {
     virtual void
     GetProjectionRaw(vr::EVREye eEye, float* pfLeft, float* pfRight, float* pfTop, float* pfBottom);
     virtual vr::DistortionCoordinates_t ComputeDistortion(vr::EVREye eEye, float fU, float fV);
+    // TODO: Implement?
```

Comment:
U can remove the comment, we just return identical stuff for the distortion anyway, just returning false is enough implementation for us.